opentelemetry-sdk/tests/resources/test_resources.py

```diff
@@ -482,6 +482,13 @@ def test_multiple_with_whitespace(self):
             detector.detect(), resources.Resource({"k": "v", "k2": "v2"})
         )
 
+    def test_invalid_key_value_pairs(self):
+        detector = resources.OTELResourceDetector()
+        os.environ[resources.OTEL_RESOURCE_ATTRIBUTES] = "k=v,k2=v2,invalid,,"
```

Comment:
previous suggestion

```suggestion
        os.environ[resources.OTEL_RESOURCE_ATTRIBUTES] = "k=v,k2=v2,invalid,,foo=bar=baz"
```

Reply:
Thanks! I'll do
```suggestion
        os.environ[resources.OTEL_RESOURCE_ATTRIBUTES] = "k=v,k2=v2,invalid,,foo=bar=baz,"
```

if that's okay with you? Because I specifically did it since I'm trying to consider the case where we have `OTEL_RESOURCE_ATTRIBUTES="new_k=new_v,$OTEL_RESOURCE_ATTRIBUTES"` in a bash script.